packages/expo-splash-screen/README.md

```diff
@@ -292,6 +292,46 @@ The newly created `SplashScreen.storyboard` needs to be marked as the `Launch Sc
 
 <img src="./assets/configuration-ios-selectLaunchScreen.png" height="350" />
 
+#### `AppDelegate.m`
+
+<!-- TODO (@bbarthec): what would happen if `expo-splash-screen` was added prior to `expo-updated` ➡️ SplashScreen registration would be doubled and unless `expo-updates` replaces rootViewController sth bad might happen 🤔 -->
```

Comment:
Recently @cruzach and I added a `priority` property to singleton modules, which should let you handle this on an infrastructure level! Basically singleton modules are sorted before `UMAppDelegateWrapper` calls the callbacks. If `EXSplashScreen` has a bigger priority than the singleton module of `expo-updates` it will always get triggered before it (at least on newer versions of `@unimodules/core`. 🙂

Reply:
Sounds good! But I think that we might not need it though 😉